playwright.config.ts

```diff
@@ -32,7 +32,7 @@ export default defineConfig({
   workers: process.env.CI ? 1 : 2,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
   // Concise 'dot' for CI, default 'list' when running locally
-  reporter: process.env.CI ? [['list'], ['html']] : 'html',
+  reporter: process.env.CI ? [['list'], ['blob']] : 'html',
```

Comment:
Use 'blob' format in CI, so that the report from each shard can be merged into a single HTML report